src/dbms/database.cpp

```diff
@@ -20,10 +22,15 @@ namespace memgraph::dbms {
 Database::Database(const storage::Config &config)
     : trigger_store_(config.durability.storage_directory / "triggers"),
       streams_{config.durability.storage_directory / "streams"} {
-  if (config.force_on_disk || utils::DirExists(config.disk.main_storage_directory)) {
+  auto storage_mode = memgraph::flags::ParseStorageMode();
```

Comment:
Is it worth while putting it into the storage::Config as `default_storage_mode`

Reply:
You mean why parse, when I can put the flag into storage::Config or?